extensions/amp-access/0.1/amp-access.js

```diff
@@ -259,9 +266,46 @@ export class AccessService {
     const expr = element.getAttribute('amp-access');
     const on = evaluateAccessExpr(expr, response);
 
-    // TODO(dvoytenko): support templates
+    /**
+     * Render templates.
+     * @type {!Array<!{template: !Element, rendered: !Promise<!Element>,
+     *    prev: ?Element}>}
+     */
+    const templates = [];
+    if (on) {
+      const templateElements = element.querySelectorAll(
```

Comment:
Done. Rearrange things a bit to ensure that, whenever possible, the element is not shown until templates are rendered.